package.json

```diff
@@ -1,110 +1,111 @@
+    "serve": "gatsby serve",
+    "lint": "eslint --fix .",
+    "pretest": "eslint --ignore-path .gitignore .",
+    "preload-fonts": "gatsby-preload-fonts",
+    "deploy": "gatsby build && gh-pages -d public -b master",
+    "noIndex": "gatsby build && echo 'User-agent: *\nDisallow: /' > public/robots.txt",
+    "prepare": "husky install"
+  },
+  "dependencies": {
+    "@fullcalendar/daygrid": "^5.10.1",
+    "@fullcalendar/google-calendar": "^5.10.1",
+    "@fullcalendar/interaction": "^5.10.1",
+    "@fullcalendar/react": "^5.10.1",
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
+    "@react-icons/all-files": "^4.1.0",
+    "axios": "^0.24.0",
+    "babel-plugin-styled-components": "^1.13.3",
+    "babel-preset-gatsby": "^2.5.2",
+    "caniuse-lite": "^1.0.30001301",
+    "dedent": "^0.7.0",
+    "eslint": "^7.32.0",
+    "formik": "^2.2.9",
+    "gatsby": "^3.13.1",
+    "gatsby-awesome-pagination": "^0.3.8",
+    "gatsby-cli": "^4.6.0",
+    "gatsby-plugin-anchor-links": "^1.2.1",
+    "gatsby-plugin-feed-mdx": "^1.1.0",
+    "gatsby-plugin-google-gtag": "^3.14.0",
+    "gatsby-plugin-image": "^1.14.2",
+    "gatsby-plugin-manifest": "^3.14.0",
+    "gatsby-plugin-mdx": "^2.14.0",
+    "gatsby-plugin-meta-redirect": "^1.1.1",
+    "gatsby-plugin-preload-fonts": "^2.14.0",
+    "gatsby-plugin-react-helmet": "^4.14.0",
+    "gatsby-plugin-react-helmet-canonical-urls": "^1.4.0",
+    "gatsby-plugin-robots-txt": "^1.7.0",
+    "gatsby-plugin-sharp": "^3.14.3",
+    "gatsby-plugin-sitemap": "^3.3.0",
+    "gatsby-plugin-styled-components": "^4.14.0",
+    "gatsby-redirect-from": "^0.4.3",
+    "gatsby-source-filesystem": "^3.14.0",
+    "gatsby-transformer-sharp": "^3.14.0",
+    "lodash": "^4.17.21",
+    "nuka-carousel": "^4.8.4",
+    "prism-react-renderer": "^1.2.1",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-accessible-accordion": "^3.3.5",
+    "react-countdown": "^2.3.2",
+    "react-countup": "^6.1.1",
+    "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
+    "react-intersection-observer": "^8.33.1",
+    "react-loadable": "^5.5.0",
+    "react-modal": "^3.14.4",
+    "react-scroll": "^1.8.4",
+    "react-select": "^3.2.0",
+    "react-slick": "^0.28.1",
+    "react-table": "^7.7.0",
+    "react-tabs": "^3.2.3",
+    "react-tooltip": "^4.2.21",
+    "react-tsparticles": "^1.39.0",
+    "react-vertical-timeline-component": "^3.5.2",
+    "react-visibility-sensor": "^5.1.1",
+    "simple-react-lightbox": "^3.6.8",
+    "slick-carousel": "^1.8.1",
+    "styled-components": "^5.3.3",
+    "swiper": "^6.8.4"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.16.8",
+    "babel-eslint": "^10.1.0",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "cpx": "^1.5.0",
+    "cross-env": "^7.0.0",
+    "env-cmd": "^10.1.0",
+    "gatsby-plugin-webpack-bundle-analyser-v2": "^1.1.26",
+    "gh-pages": "^3.2.3",
+    "husky": "^7.0.4",
+    "lint-staged": "^12.3.3",
+    "rimraf": "^3.0.2"

```

Comment:
Is this package newly added?
If so, for what feature?

Reply:
I think it's from the conflicts resolve merge cause I just added the `react-countdown` package :thinking:

Reply:
Ahh...cool. It was added after the husky update. No worries,